uint8 status
# 0 -> Aborted
# 1 -> Completed
# 2 -> Timeout
